<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Abinash" failures="0" tests="3" name="vTigerCRMappPratice.DataProviderException" time="41.143" errors="0" timestamp="2023-01-24T12:52:41 IST" skipped="0">
  <testcase classname="vTigerCRMappPratice.DataProviderException" name="login" time="13.669"/>
  <system-out/>
  <testcase classname="vTigerCRMappPratice.DataProviderException" name="login" time="13.317"/>
  <system-out/>
  <testcase classname="vTigerCRMappPratice.DataProviderException" name="login" time="14.157"/>
  <system-out/>
</testsuite> <!-- vTigerCRMappPratice.DataProviderException -->
